src/app.js

```diff
@@ -0,0 +1,3 @@
+import app from './server';
```

Comment:
this file, should be merged in src/bin/www

Reply:
IMO purpose of `./bin/www` is to proxy source code to babel and should not do anything more, while `app.js` is the app launch it self.